docs/pypi.md

```diff
@@ -33,10 +33,39 @@ finished loading:
 
 ```
 def do_work(*args):
-  import snowballstemmer
-  snowballstemmer.stemmer('english')
-  stemmer.stemWords('go goes going gone'.split())
+    import snowballstemmer
+    stemmer = snowballstemmer.stemmer('english')
+    print(stemmer.stemWords('go goes going gone'.split()))
 
 import micropip
 micropip.install('snowballstemmer').then(do_work)
 ```
+
+## A Minimal working example
+
+```html
+<html>
+<head>
+  <script type="text/javascript" src="https://pyodide.pymedphys.com/pyodide.js"></script>
```

Comment:
It is problematic to use a custom domain in the docs. Using `https://alpha.iodide.app/pyodide-0.12.0/` does not seem to work either but it would be good to find some solution.

Reply:
That's a bit out of my control I think. It doesn't look like iodide app allows CORS. Here is the repo I used to create that custom domain:

https://github.com/pymedphys/netlify-pyodide

Potentially something like "https://cdn.iodide.app" could point to a fork of that repo?

Reply:
I'll leave this for someone else to solve if that is okay?